papis/commands/add.py

```diff
@@ -315,6 +313,7 @@ def run(paths: List[str],
 
     if not folder_name:
         out_folder_name = get_hash_folder(data, in_documents_paths)
+        out_folder_path = out_folder_path / out_folder_name
```

Comment:
`out_folder_name` isn't used anywhere after this point anymore.

Reply:
Good one!